bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/net/NetworkAddressChangeListener.java

```diff
@@ -39,4 +41,13 @@
      */
     void onChanged(List<CidrAddress> added, List<CidrAddress> removed);
 
+    /**
+     * When the primary address is changed, listeners will be notified by this method. The primary address may be
+     * changed either through the PaperUI or through an update to the associated configuration file. The primary address
```

Comment:
Please remove the reference to paperUI here. The configuration can be updated through any method.